index.html

```diff
@@ -0,0 +1,176 @@
+                    
+                    <div class="forimg">
+                        <a class="yicon" href="https://www.youtube.com">
+                            <img class="imgspan" src="Images/yicon.png" width="35" height="35" />
+                            <span>YouTube<sup>NG</sup></span>
+                        </a>
+                    </div>
+                    
+                    <div class="forform">
+                        <form class="header-form">
+                            <input class="search" type="search" name="search" value="Search" placeholder="Search" />
+                            <span class="scbtn"><button type="submit"><i class="fa fa-search"></i></button></span>
+                        </form>
+                    </div>
+                    
+                    <div class="microphone">
+                        <div class="fa-mic">
+                            <i class="fa fa-microphone" style="font-size: 20px; color: black;"></i>
+                        </div>
+                    </div>
+
+                    <div class="video-icon-div">
+                        <div class="vid-icon">
+                            <i class="material-icons">video_call</i>
+                        </div>
+                    </div>
+
+                    <div class="ytb-app-div">
+                        <div class="ytb-app-icon">
+                            <i class="material-icons">apps</i>
+                        </div>
+                    </div>
+
+                    <div class="bell-icon-div">
+                        <div class="bell-icon">
+                            <i class="fa fa-bell" style="font-size: 20px;"></i>
+                        </div>
+                    </div>
+
+                    <div class="profile-div">
+                        <div class="profile-pic"></div>
+                    </div>
+                    
+                </div>
+        </header>
+
+        <main>
+            <section class="vid-section">
+                <div class="iframe-div">
+                    <iframe class="iframes" src="https://www.youtube.com/embed/KqG-FWWjn3g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+
+                <div>
+                    <h1 class="h1-title">Clarification of Dr Sharaf On the Issue of his daughter's wedding</h1>
+                </div>
+
+                <div class="likes-views-div">
+                    <div class="views-date">
+                        <span class="views">135 views<span class="views-dot">.</span>Jan 29, 2022</span>
+                    </div>
+
+                    <div class="likes-share-save">
+                        <div class="like-div likes">
+                            <i class="fa fa-thumbs-up" style="font-size: 20px;"><span style="font-style: bold; font-family:'Times New Roman', Times, serif"> 5</span></i>
```

Comment:
- [ ] You have used polite words but kindly answer the student why they want these changes by explaining why it is better.